### ISSUE TITLE
Issues on docker

### ISSUE THREAD

**Issue by karan** [sindresorhus/pageres#175]
Long story short, without docker everything is :+1: 

With docker, I get this weird error for the URL `http://web.archive.org/web/20041203145812/http://www.twitter.com:80/`.

```bash
worker_1 | events.js:85
worker_1 |       throw er; // Unhandled 'error' event
worker_1 |             ^
worker_1 | Error: /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: 2: /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: ��ըa�: not found
worker_1 | /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: 1: /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: �~*@��$/�?�c�
                                                                                                                          /usr/lib/��: not found
worker_1 | /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: 3: /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: 2: /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: �: not found
,3�,3��|�����������݅8__PAGEZERO��o$wTEXTg�v�K{�^
                                                =^text]`��C�V(="�F����Z�gconst
worker_1 | al���m��~б����symbol
                               tub۵��E#�5�^��x_helper�|ZR��v�)J�l/ވring�v�`��6�t��p�[fgcc_�cep(tab(����
                                                                                                       +xBb�*��(6,�/�M0bM�-t7H=���_�fo_plist��uEh8b��uu�uŚ�=`�vv��A;nw�d/?P���P0veh_frm�	�ameжv09|Ц{���G�wDATA1��>]�>pO�?6T7	�dyld�MS
��(jqئ                                                 ���)nl�*r�Z�>�$���6�w�la����w�
      �moi2fֵG�*c��"v��&	�da����b`#v�z`w��� ����y�� hh��y�f9Z�L̆n�Q���L��/bss�qL�]�v�j-����`�m�n��A��9�T 4|۵wwOBJC1`v0��LcC�7�"r��
                                                                                                                                �age_refs�ҁ�!�sjo~ls��gNh]�Ǉ1s��9���h�Ȩ�룵Q�6ule>͉�O�Lj�L�VQH�a�|�Їlx�CKe�����D��n��s�h�k
                                   �q�|ч9��at��8r����҇proto@�Cls��Ӛ`�_W�PA�4tpԗ
                                                                              -���st�����a�h�jBvars��|Rjo@܇1o;*��H
                                                                                                                  ߗ:��egory�4��AA���mags��FՇ��ǉWLINKEDI߇�&�v,C����[\ןh�N�~�_��ɉXiP�������J����=: Protocol error
worker_1 | /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: 6: /worker/node_modules/pageres/node_modules/screenshot-stream/node_modules/phantom-bridge/node_modules/phantomjs/lib/phantom/bin/phantomjs: Syntax error: "(" unexpected
worker_1 |     at Socket.<anonymous> (/worker/node_modules/pageres/node_modules/screenshot-stream/index.js:57:14)
worker_1 |     at Socket.emit (events.js:107:17)
worker_1 |     at readableAddChunk (_stream_readable.js:163:16)
worker_1 |     at Socket.Readable.push (_stream_readable.js:126:10)
worker_1 |     at Pipe.onread (net.js:538:20)
worker_1 | 
```

Do I need to expose some ports?

**Comment by sindresorhus** [comment]
Seems like the PhantomJS binary is either not found or corrupt. Try reinstalling? Not really sure how to help you as I've never used docker and I have no way to reproduce.

**Comment by kevva** [comment]
@karan, did you ever get this working?

**Comment by SamVerschueren** [comment]
It's weird this is failing, the tests are ran with Docker as well (travis) and when I run the tests with [trevor](https://github.com/vdemedes/trevor) everything seems to work.

**Comment by SamVerschueren** [comment]
@karan Care to share your `Dockerfile`?

**Comment by karan** [comment]
```
FROM node:0.12

RUN mkdir /worker
WORKDIR /worker

ADD package.json /
RUN cd /; npm install

COPY . /worker/
```

**Comment by SamVerschueren** [comment]
@karan your other files might be usefull as well to reproduce the error. package.json and index.js will do it.

**Comment by SamVerschueren** [comment]
For the sake of testing this, I create a small docker image to generate the images instead of `pageres-cli`. It's just very basic.

> TLDR; It works

#### Dockerfile
```Dockerfile
FROM	node:0.12

COPY	. /app

RUN		cd /app; npm install --production

ENTRYPOINT 	["node", "/app/index.js"]
```

#### index.js
```js
'use strict';
var Pageres = require('pageres');

if (process.argv.length === 2) {
	console.error('Please provide a URL');
	process.exit(1);
}

new Pageres()
	.src(process.argv.pop(), ['1024x768'])
	.dest('/workdir')
	.run();
```

#### Usage

##### build
```
$ docker build -t pageres .
```

##### execute
```
$ docker run -v /Path/to/store/images/:/workdir pageres yeoman.io
$ docker run -v /Path/to/store/images/:/workdir pageres http://web.archive.org/web/20041203145812/http://www.twitter.com:80/
```

### Conclusion
It works correctly and renders the page as it should be. So I don't think this is a `pageres` issue. Try to rebuild your image, maybe something went wrong with the `phantomjs` installation.

--------
@sindresorhus @kevva Should we close this?

**Comment by SamVerschueren** [comment]
Closing this due to inactivity and unreproducible
